app/javascript/plugins/progress-tracker.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+import $ from 'jquery';
+import ee from '../shared/pub_sub';
+
+export default class ProgressTracker {
+  constructor() {
+    this.addEventListeners();
+  }
+
+  addEventListeners() {
+    ee.on('action:submitted_success', this.show);
+    ee.on('action:submitted_success', () => this.tick('signed'));
+    ee.on('fundraiser:transaction_success', () => this.tick('donated'));
+    window.addEventListener(
+      'share',
+      () => this.addEventListeners.tick('shared'),
+      false
+    );
+    $('.two-step__decline').on('click', () => {
+      this.cross('shared');
+    });
+  }
+
+  show() {
+    const $tracker = $('.header-logo .progress-tracker');
+    $tracker
+      .removeClass('hidden-closed')
+      .clone()
```

Comment:
What's the clone for?

Reply:
So basically, we need to make a copy of the tracker div and place it in the mobile overlay. If we don't clone it, `prependTo` will move it (so it disappears from the header).

Reply:
:+1: